.typos.toml

```diff
@@ -0,0 +1,16 @@
+# https://github.com/crate-ci/typos
+# install:  cargo install typos-cli
+# run:      typos
+
+[default.extend-words]
+# There are some weird words in this repository:
```

Comment:
uhm you're sure about this? Some of those looks like straight up typos

Reply:
I am in fact not sure, but I fixed all the typos I dared fix 😅